lib/ansible/modules/yum_repository.py

```diff
@@ -24,7 +24,6 @@
       - If set to C(yes) Yum will download packages and metadata from this
         repo in parallel, if possible.
     type: bool
-    default: 'yes'
```

Comment:
this should make a note that the default has changed making it backwards incompatible

Reply:
As far as I can tell, this was the default behavior until 3e9943bc5e (which was first included in Ansible 2.11).

Before then, the documentation stated that the default value was `yes`, but the code stated

```python
argument_spec['async'] = dict(type='bool')
```

So how should I document this? Should I add a comment in the description, or is there a more specific field for that?

It's not clear to me if users relied on the documented behavior until now (even though it didn't behave this way until 2.11), or if this commit is actually rectifying a breaking change that (as far as I can tell) hadn't been documented.

Reply:
hmm, had not realized that, probably fine to just remove then

Reply:
So in its current state, this PR can be backported to 2.11 and 2.12 without any changes.

If it gets backported to 2.10 as well (but isn't it receiving only security updates by now), then it would only include the documentation change, since the code is already in this state (i.e. no default value).

Or did you have in mind some other place where the documentation needs updating?

Reply:
@bcoca Did you want me to make any changes to this PR?